src/main/java/org/owasp/benchmark/score/report/ScatterPlot.java

```diff
@@ -0,0 +1,246 @@
+        rangeAxis.setTickUnit(new NumberTickUnit(10));
+        rangeAxis.setAxisLineVisible(true);
+        rangeAxis.setMinorTickMarksVisible(true);
+        rangeAxis.setTickMarksVisible(true);
+        rangeAxis.setLowerMargin(10);
+        rangeAxis.setUpperMargin(10);
+        
+        domainAxis.setRange(-5, 175);
+        domainAxis.setNumberFormatOverride(pctFormat);
+        domainAxis.setTickLabelPaint(Color.decode("#666666"));
+        domainAxis.setMinorTickCount(5);
+        domainAxis.setTickUnit(new NumberTickUnit(10));
+        domainAxis.setAxisLineVisible(true);
+        domainAxis.setTickMarksVisible(true);
+        domainAxis.setMinorTickMarksVisible(true);
+        domainAxis.setLowerMargin(10);
+        domainAxis.setUpperMargin(10);
+        
+        xyplot.setRangeGridlineStroke(new BasicStroke());
+        xyplot.setRangeGridlinePaint(Color.lightGray);
+        xyplot.setRangeMinorGridlinePaint(Color.decode("#DDDDDD"));
+        xyplot.setRangeMinorGridlinesVisible(true);
+        xyplot.setOutlineVisible(true);
+        xyplot.setDomainGridlineStroke(new BasicStroke());
+        xyplot.setDomainGridlinePaint(Color.lightGray);
+        xyplot.setDomainMinorGridlinePaint(Color.decode("#DDDDDD"));
+        xyplot.setDomainMinorGridlinesVisible(true);
+        xyplot.getRenderer().setSeriesPaint(0, Color.decode("#4572a7"));
+
+        chart.setTextAntiAlias(true);
+        chart.setAntiAlias(true);
+        chart.removeLegend();
+        chart.setPadding(new RectangleInsets(20, 20, 20, 20));
+        
+        Point2D legendLocation = new Point2D.Double( 101, -10 );
+        makeRect(xyplot, legendLocation, 120, 74, Color.WHITE );
+       
+        Point2D triangleLocation = new Point2D.Double( 101, -10 );
+        Color grey = new Color(0.1f,0.1f,0.1f,0.1f);
+        makeTriangle(xyplot, triangleLocation, grey );
+        
+        makeGuessingLine( xyplot );
+    }
+    
+    
+    public static XYPointerAnnotation makePointer(int x, int y, String msg, TextAnchor anchor, int angle ) {
+        XYPointerAnnotation pointer = new XYPointerAnnotation(msg, x, y, Math.toRadians(angle));
+        pointer.setBackgroundPaint(Color.white);
+        pointer.setTextAnchor(anchor);
+        pointer.setArrowWidth(4);
+        pointer.setArrowLength(8);
+        pointer.setArrowPaint(Color.red);
+        pointer.setLabelOffset(2);
+        pointer.setPaint(Color.red);
+        pointer.setFont(theme.getRegularFont());
+        return pointer;
+    }
+    
+    public void writeChartToFile(File f, int height) throws IOException {
+        FileOutputStream stream = new FileOutputStream(f);
+        ChartUtils.writeChartAsPNG(stream, chart, (int)Math.round(height*1.4), height);
+        stream.close();
+    }
+
+    public static void addGenerationDate(XYPlot xyplot) {
+        //add scorecard generation date
+        final String pattern = "dd MMM yyyy h:mm a";
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.